common/test_fiximports.py

```diff
@@ -2,12 +2,16 @@
 from textwrap import dedent
 from twisted.trial import unittest
 
+#returning functions
+def d_setUp():
+    return None
 
 class TestFixImports(unittest.TestCase):
 
     def setUp(self):
         self.fiximports = FixImports()
-        self.fiximports.printErrorMsg = lambda *_: None
+        ret_msg = d_setUp()
```

Comment:
Note that the existing code assigned a function of any number of positional arguments that always returns None.  That's very different from the simple value None.